libretroshare/src/retroshare/rsinit.h

```diff
@@ -51,6 +51,15 @@ struct RsLoginHelper;
  */
 extern RsLoginHelper* rsLoginHelper;
 
+
+class RsAccounts;
+
+/**
+ * Pointer to global instance of RsAccounts needed to expose JSON API
+ * @jsonapi{development}
+ */
+extern RsAccounts* rsAccounts;
```

Comment:
You shouldn't need this: RsAccounts only has static members (I made sure of it because I want to simplify the code so I started with making this classs static). So if possible convert the getCurrentAccountId() into a static method. thx!

Reply:
It is needed by JSON API because the code is automatically generated and because almost all services uses this "wild pointer" pattern the code generator assumes the pointer exists to call the methods, but I'll make getCurrentAccountId static so it can be used without the pointer in the other parts of the code